project/Build.scala

```diff
@@ -853,10 +857,16 @@ object Build extends sbt.Build {
 
           case _: PhantomJSEnv =>
             Seq(Tests.Argument("-tphantomjs"))
+
+          case env: RetryingComJSEnv =>
+            envTagsFor(env.baseEnv)
+
           case _ =>
```

Comment:
Yep, good idea. I'll first see whether the CI is going anywhere with the current commit.